New main (new work with corrections)

pythonstudent01 opened the pull request:
This is my second attempt at creating the asciidoc files. Please use the giteditor to view the text files to see the syntax I was using to create these files. I don't know how to demonstrate my changes here.

ragevou commented on homework-projects/CharlesDube/Index index.adoc:
```diff
@@ -0,0 +1,14 @@
+
+*Replace me:* This index file template contains two included files listed below.
+
+include::Draft Part 1.adoc[leveloffset=+1]
```
include is not building due to incorrect file name.

ragevou commented on homework-projects/CharlesDube/Index index.adoc:
```diff
@@ -0,0 +1,14 @@
+
+include::Draft Part 1.adoc[leveloffset=+1]
+
+include::Draft Part 2.adoc[leveloffset=+1]
```
include is not building due to incorrect file name.

ragevou commented on homework-projects/CharlesDube/Index index.adoc:
```diff
@@ -0,0 +1,14 @@
+
+include::Draft Part 2.adoc[leveloffset=+1]
+
+include::https://media.tenor.com/OqvDDg1mwCMAAAAC/i-dont-know-what-the-hell-im-doing-matt-logelin.gif[leveloffset=+1]
```
incorrect syntax for including a link. See more details: https://docs.asciidoctor.org/asciidoc/latest/macros/links/

ragevou commented on homework-projects/CharlesDube/PhotoColage Part 2.adoc:
```diff
@@ -0,0 +1,20 @@
+[[HowToUse]]
+=== How to use
+
+1. Download the installation files from link:https://github.com/adrienverge/PhotoCollage[PhotoCollage Repository] 2.1. On Fedora run 'sudo dnf install PhotoCollage' in the terminal 2.2. On Debian 9+ run 'sudo apt-get install photocollage' 2.3. On pip Python manager run 'sudo pip3 install photocollage'
```
it's an incorrect asciidoc syntax for numbered list. See more details: https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/

pythonstudent01 commented:
I also sent an email via IS to Sarka. I hope that all should be clear. Thank you.

ragevou commented:
the new-main branch is not building locally. 

<img width="1003" alt="Screenshot 2023-04-20 at 15 35 13" src="https://user-images.githubusercontent.com/3250677/233384605-adc053d8-10fe-4627-95bf-ac194cd802d2.png">

pythonstudent01 commented:
Greetings again, I found some spelling errors (that were in relation to links). These spelling errors didn't allow any link to PhotoCollage Part 1 document to work. Maybe this is what caused the errors when building the document. Please may you try again and it should work now.

pythonstudent01 commented:
I've removed the image of the Tea from the file and I have removed the Gif. When I click the links I travel between files without problems, I don't know what's happening on your end? You are viewing the file on GitHub?

domiborges commented:
Closing as outdated
